quartermaster/src/main/java/com/github/reflectoring/Quartermaster.java

```diff
@@ -12,4 +16,21 @@ public static void main(String[] args) {
         new SpringApplicationBuilder().sources(Quartermaster.class).run(args);
     }
 
+    /*
+    * Activates Cross-Origin Resource Sharing (CORS)
+    * for local development of JavaScript client.
+    *
+    * https://de.wikipedia.org/wiki/Cross-Origin_Resource_Sharing
+    */
+    @Bean
+    public WebMvcConfigurer corsConfigurer() {
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Document this public method. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AUndocumentedApi)